themes/default/src/webviz_default_theme/templates/main.html

```diff
@@ -15,7 +15,7 @@
 <body class="Ff(de)">
     <div class="D(f)--sm H(100vh)">
         {% include 'menu.html' %}
-        <main class="D(f) Fxd(c) Fxg(1)--sm H(100vh) Ovy(a)--sm Ovs(touch) Scb(s)">
+        <main class="D(f) Fxd(c) Fxg(1)--sm H(100vh) Ovy(a)--sm Ovs(touch) Scb(s) Mstart(20px) Mend(20px)">
```

Comment:
@fortena Should the margin be added after `call(banner)`, and instead around `{{ webviz.content() }}` alone?

I'm not 100% sure, have not tried to the current banner implementation. But if is like the image+title at the top of https://www.equinor.com/ it is probably visually better to not add margin around the banner image, but only around the content below it.